app/controllers/users_controller.rb

```diff
@@ -19,14 +22,53 @@ def create
       reset_session
       log_in @user
       flash[:success] = "Welcome to the Food Universe!"
-      redirect_to @user
+      if @user.admin?
```

Comment:
This if/else could be a 1-liner. Also, no need for the `elsif @user.user?`, because you only 2 user types. In case an extra user type comes along, you wouldn't want multiple `elsifs` anyway, you will use a switch statement

```ruby
@user.admin? ? redirect_to admin_dashboard_path : redirect_to user_path(@user)
```